source/client/benchmark_client_impl.cc

```diff
@@ -82,8 +82,8 @@ bool BenchmarkClientHttpImpl::tryStartRequest(CompletionCallback caller_completi
   // exactly as possible here.
   // NOTE: We can't consistently rely on resourceManager()::requests()
   // because that isn't used for h/1 (it is used in tcp and h2 though).
-  if ((max_pending_requests_ == 1 &&
-       (requests_initiated_ - requests_completed_) >= connection_limit_)) {
+  if (max_pending_requests_ == 1 &&
+      ((requests_initiated_ - requests_completed_) >= connection_limit_)) {
```

Comment:
Nit: these parens are redundant.

Reply:
Backed out this change, it was included accidentally.